resources/migrations/instance_analytics_views/h2-indexes.sql

```diff
@@ -0,0 +1,7 @@
+drop index if exists idx_view_log_timestamp;
```

Comment:
I've changed the create & drop index commands to use `if exists` and `if not exists` across the board. Shouldn't be necessary but just wanted to be extra defensive.

Reply:
(Removing them from MySQL; didn't realize the syntax is not supported)